draft-jones-perc-dtls-tunnel/draft-jones-perc-dtls-tunnel.md

```diff
@@ -496,7 +508,7 @@ struct {
 ```
 
 The fields are described as follows:
-* association_id: The value of the association identifier used to uniquely identify each endpoint in a conference
+* association_id: The value of the association identifier used to uniquely identify each endpoint connected to the media distributor
```

Comment:
Right.
I actually changed this before reading the To-Do List section further below. So my point is that the KD currently has no idea of what a conference is.
As the MD chooses the association ID I guess it "only" represents a unique endpoint from the MD's perspective.